src/app/models/session.ts

```diff
@@ -17,12 +18,13 @@ export class Session {
     this.sessionName = sessionName;
     this.status = SessionStatus.inactive;
     this.startDateTime = undefined;
+    this.duration = environment.sessionDuration;
     this.region = region;
   }
 
   expired(): boolean {
     const currentTime = new Date().getTime();
     const startTime = new Date(this.startDateTime).getTime();
-    return (currentTime - startTime) / 1000 > environment.sessionDuration;
+    return (currentTime - startTime) / 1000 > this.duration;
```

Comment:
If `this.duration` is unset, then shouldn't it use the `environment.sessionDuration` portion ?

Reply:
This PR should probably be closed- it was open prior to the 0.9.0 release which was a pretty substantial change. #283 addresses what this PR was meant to achieve, but work should probably start over in a new branch.